provisioner/terraform/serve.go

```diff
@@ -16,9 +16,9 @@ import (
 
 // This is the exact version of Terraform used internally
 // when Terraform is missing on the system.
-var terraformVersion = version.Must(version.NewVersion("1.1.9"))
+var terraformVersion = version.Must(version.NewVersion("1.2.1"))
 var minTerraformVersion = version.Must(version.NewVersion("1.1.0"))
-var maxTerraformVersion = version.Must(version.NewVersion("1.2.0"))
+var maxTerraformVersion = version.Must(version.NewVersion("1.2.1"))
```

Comment:
As per the logic in line 70, we are checking that the installed version should be strictly less than the max version. This would throw an error for 1.2.1 and then install 1.2.1 again. The idea behind min and max range was to allow the same minor version.  Maybe we can update the range to 1.2.0 <= version < 1.3.0?

Reply:
Ah, nice catch. I don't know about you, but "max" to me implies that the maximum is still allowed. 
How about we change the logic on line 70 to 
```
version.LessThan(minTerraformVersion) || version.GreaterThan(maxTerraformVersion)
```

Thoughts?